tasks/get_current_archive_list/README.md

```diff
@@ -14,142 +18,4 @@ Visit the [Developer Guide](https://nasa.github.io/cumulus-orca/docs/developer/d
-    
-    truncate_s3_partition(orca_archive_location: str, engine: sqlalchemy.future.engine.Engine) -> None
-        Truncates the partition for the given orca_archive_location, removing its data.
-        
-        Args:
-            orca_archive_location: The name of the bucket to generate the reports for.
-            engine: The sqlalchemy engine to use for contacting the database.
-    
-    truncate_s3_partition_sql(partition_name: str)
-    
-    update_job_sql()
-    
-    update_job_with_failure(job_id: int, error_message: str, engine: sqlalchemy.future.engine.Engine) -> None
-        Updates the status entry for a job.
-        
-        Args:
-            job_id: The id of the job to associate info with.
-            error_message: The error to post to the job.
-            engine: The sqlalchemy engine to use for contacting the database.
-    
-    update_job_with_s3_inventory_in_postgres(s3_access_key: str, s3_secret_key: str, orca_archive_location: str, report_bucket_name: str, report_bucket_region: str, csv_key_paths: List[str], manifest_file_schema: str, job_id: int, engine: sqlalchemy.future.engine.Engine) -> None
-        Deconstructs a record to its components and calls send_values_to_database with the result.
-        
-        Args:
-            s3_access_key: The access key that, when paired with s3_secret_key, allows postgres to access s3.
-            s3_secret_key: The secret key that, when paired with s3_access_key, allows postgres to access s3.
-            orca_archive_location: The name of the bucket to generate the reports for.
-            report_bucket_name: The name of the bucket the csv is located in.
-            report_bucket_region: The name of the region the report bucket resides in.
-            csv_key_paths: The paths of the csvs within the report bucket.
-            manifest_file_schema: The string representing columns present in the csv.
-            job_id: The id of the job to associate info with.
-            engine: The sqlalchemy engine to use for contacting the database.
-
-DATA
-    Any = typing.Any
-    BUCKET_NAME_KEY = 'name'
-    Dict = typing.Dict
-    EVENT_RECORDS_KEY = 'Records'
-    FILES_KEY_KEY = 'key'
-    LOGGER = <cumulus_logger.CumulusLogger object>
-    List = typing.List
-    MANIFEST_CREATION_TIMESTAMP_KEY = 'creationTimestamp'
-    MANIFEST_FILES_KEY = 'files'
-    MANIFEST_FILE_SCHEMA_KEY = 'fileSchema'
-    MANIFEST_SOURCE_BUCKET_KEY = 'sourceBucket'
-    OBJECT_KEY_KEY = 'key'
-    OS_S3_ACCESS_KEY_KEY = 'S3_ACCESS_KEY'
-    OS_S3_SECRET_KEY_KEY = 'S3_SECRET_KEY'
-    OUTPUT_JOB_ID_KEY = 'jobId'
-    RECORD_AWS_REGION_KEY = 'awsRegion'
-    RECORD_S3_KEY = 's3'
-    S3_BUCKET_KEY = 'bucket'
-    S3_OBJECT_KEY = 'object'
-    column_mappings = {'Bucket': 'orca_archive_location text', 'ETag': 'et...
-    raw_schema = <_io.TextIOWrapper name='schemas/output.json' mode='r' en...
-```
+See API.md for documentation.
```

Comment:
I would do this as a link `[API.md](See the API documentation for more details.)`